portfolio/src/main/webapp/script.js

```diff
@@ -48,6 +51,19 @@ function OnWindowScrolled() {
     }
 }
 
+/**
+ * Callback when comments-container is scrolled
+ */
+function OnCommentsContainerScrolled() {
+    // Get the comments container element
+    const commentsElement = document.getElementById('comments-container');
+
+    // If commentsElement was scrolled to the top, load more comments
```

Comment:
I can't quite see it. When the page is loaded, the element is not scrolled to the top? In my mind it is scrolled to the top, and it is constantly reloading new comments. Please explain how it is actually working (if you can do it in the comment, even better).

Reply:
Because the comments-container has a `display:flexbox` and a `flex-direction:column-reverse`, the browser sees it upside down and it will be scrolled to the bottom when the page is loaded. When you scroll it by hand to the top, if there are more comments to load, it will make the div bigger in height (because it adds more comments) and it will be no longer scrolled to the top. I tested it, it works pretty good.

I will try to explain that in the comment also.

Reply:
Got it, thanks.